R-packages/evalcast/R/plot_trajectory.R

```diff
@@ -97,25 +101,49 @@ plot_trajectory <- function(list_of_predictions_cards,
   response_df <- response_df %>% dplyr::mutate(
     location_abbr = if(geo_type == "state") covidcast::fips_to_abbr(paste0(location,"000")) else covidcast::fips_to_name(location)) %>%
     dplyr::filter(!is.na(location_abbr))
```

Comment:
This is also pretty ugly. It runs over a lot and is hard to read. Can we clean this up?

Reply:
Okay, I'll look over the script and fix the indenting issue:)